tensorflow/lite/delegates/gpu/metal_delegate.h

```diff
@@ -67,6 +68,17 @@ TFL_CAPI_EXPORT extern TfLiteDelegate* TFLGpuDelegateCreate(
 // Destroys a delegate created with `TFLGpuDelegateCreate` call.
 TFL_CAPI_EXPORT extern void TFLGpuDelegateDelete(TfLiteDelegate* delegate);
 
+// Binds Metal buffer to an input or an output tensor in the initialized
+// delegate. Bound buffer should have sufficient storage to accommodate all
+// elements of a tensor. For quantized model, the buffer is bound to internal
+// dequantized float32 tensor.
+// Returns non-zero on success, or zero otherwise.
+//
+// *** Must be called *after* `Interpreter::ModifyGraphWithDelegate`. ***
```

Comment:
Please add a WARNING that this is experimental, so that its not taken to be a stable feature :-)

Reply:
+1

Reply:
Added in ef4ddc6700f4c84096b977aa963d99199a5cbbcb